packages/tooltip/src/Tooltip.story.js

```diff
@@ -1,137 +1,249 @@
         <Tooltip
-          align={alignments.AUTO}
-          text="This is an auto aligned tooltip"
+          trigger={<Button type={buttonTypes.SECONDARY}>Hover me</Button>}
+          content="Auto aligned/positioned tooltip"
+          positionX={POSITIONS.X.AUTO}
+          positionY={POSITIONS.Y.AUTO}
         />
       </div>
-    </div>
-  ))
-  .add("auto aligned (with bondaries)", () => (
-    <div
-      style={{
-        border: "solid 1px orangered",
-        display: "flex",
-        justifyContent: "space-between",
-        width: "600px",
-      }}
-      className="exampleContainer"
-    >
-      <div style={{ position: "relative", width: "50px", border: "1px solid" }}>
-        Child
+
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+          width: "100%",
+          position: "fixed",
+          bottom: 0,
+        }}
+      >
         <Tooltip
-          align={alignments.AUTO}
-          text="This is an auto aligned tooltip"
-          bondariesSelector=".exampleContainer"
+          trigger={<Button type={buttonTypes.SECONDARY}>Hover me</Button>}
+          content="Auto aligned/positioned tooltip"
+          positionX={POSITIONS.X.AUTO}
+          positionY={POSITIONS.Y.AUTO}
         />
-      </div>
-      <div style={{ position: "relative", width: "50px", border: "1px solid" }}>
-        Child
         <Tooltip
-          align={alignments.AUTO}
-          text="This is an auto aligned tooltip"
-          bondariesSelector=".exampleContainer"
+          trigger={<Button type={buttonTypes.SECONDARY}>Hover me</Button>}
+          content="Auto aligned/positioned tooltip"
+          positionX={POSITIONS.X.AUTO}
+          positionY={POSITIONS.Y.AUTO}
         />
       </div>
     </div>
   ))
-  .add("custom padding", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip
-        isVisible
-        text="Left aligned with custom padding"
-        padding="30px"
-      />
-    </div>
-  ))
-  .add("custom top", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip isVisible text="Left aligned with custom top" top="30vh" />
-    </div>
-  ))
-  .add("with content overflow hidden", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip isVisible padding="0" overflow="hidden">
-        <div style={{ background: "aliceblue" }}>It should not overflow</div>
-      </Tooltip>
-    </div>
-  ))
-  .add("extended style", () => (
-    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
-      <Tooltip
-        css="
-          text-decoration-line: underline overline;
-        "
-        isVisible
-        text="This is a left aligned tooltip"
-      />
-    </div>
-  ));
+  .add("auto aligned/positioned (with boundaries)", () => {
```

Comment:
The auto position is not working here.
![2019-07-18 10 05 22](https://user-images.githubusercontent.com/1459283/61465113-d0abb280-a94d-11e9-8e95-345a325772f7.gif)